src/System.Data.SqlClient/tests/StressTests/IMonitorLoader/IMonitorLoader.csproj

```diff
@@ -5,8 +5,7 @@
     <ProjectGuid>{AF78BA88-6428-47EA-8682-442DAF8E9656}</ProjectGuid>
     <RootNamespace>Monitoring</RootNamespace>
     <AssemblyName>Monitoring</AssemblyName>
-    <OutputType>Library</OutputType>
-    <NugetTargetMoniker>.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <IsTestProject>true</IsTestProject>
```

Comment:
We should probably still change IsTestProject to look for the "/tests/" in the path.